back-end-side/Models/Record.cs

```diff
@@ -18,6 +18,9 @@ public class Record : IComparable
         [Ignore]
         public string? Category { get; set; } = "other";
 
+        [Ignore]
+        public bool IsCategorized { get; set; }
```

Comment:
Maybe it would be good to initialize _false_ as default value of ``isCategorized``?

Reply:
Yes, it would, thanks